play-json/src/main/scala/com/lightbend/lagom/scaladsl/playjson/SerializerRegistry.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package com.lightbend.lagom.scaladsl.playjson
+
+import scala.collection.immutable.Seq
```

Comment:
I prefer to import `scala.collection.immutable` and use `immutable.Seq` as the type (imports can easily be lost during refactoring)